main.js

```diff
@@ -229,7 +229,7 @@ function init(vertSource, fragSource) {
 
 
     var text = { Model: "BoomBox" };
```

Comment:
What do you think about making this model the default one upon opening the demo? I think it displays more of a variety of materials than the `BoomBox` model (mirroring glass, brushed metal sides, smoother metal on back, metallic/rough scratches with interesting highlights, and diffuse tubing underneath).